avro/src/main/java/com/fasterxml/jackson/dataformat/avro/deser/AvroUUIDDeserializer.java

```diff
@@ -0,0 +1,18 @@
+package com.fasterxml.jackson.dataformat.avro.deser;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public class AvroUUIDDeserializer extends JsonDeserializer<UUID> {
```

Comment:
Is there need to add this? Wouldn't standard `UUIDDeserializer` from `jackson-databind` work?
(it actually supports both textual and 16-byte binary value)

Reply:
After playing with this a bit on 2.10, it looks like default serialization of `java.util.UUID` with Avro is unfortunately `String`. I can work on changing that in 2.11, but for 2.10 this would actually make sense I guess.